include/core/CFastMutex.h

```diff
@@ -45,16 +45,10 @@ namespace core {
 //!
 //! All errors are just warnings - no action taken.
 //!
-//! On Mac OS X, spin locks are also used, which might seem strange
-//! given that Mac OS X will have relatively few available CPU cores
-//! (say 2-4).  However, Mac OS X spinlocks are actually quite clever
-//! about yielding if they don't quickly get a lock, and are hence
-//! more akin to adaptive mutexes than pure spinlocks.  The source
-//! code at confirms this:
-//! http://www.opensource.apple.com/source/Libc/Libc-825.25/x86_64/sys/spinlocks_asm.s
-//! The other reason is that pthread mutexes are appallingly slow on
-//! Mac OS X so it's easy to massively outperform them - e.g. see:
-//! http://www.mr-edd.co.uk/blog/sad_state_of_osx_pthread_mutex_t
```

Comment:
I think it's worth keeping this bit about why we don't just use pthread_mutex_t on macOS

Reply:
Sure, sadly that blog post no longer exists though :-/